CDCgov/prime-reportstream
# SFTP Cron Job Azure Function

This PR adds a cron job that will check all the remote connections for receivers and logs the results to a new DB table that will let us check and make sure that our ability to connect to the remote locations for receivers.

This is to be used a replacement for the `ALL` check that can only be done on the command line currently. We have too many receivers at this point and the `ALL` check fails every time it runs. We can no longer get a holistic view of all of the receivers up/down status, and there are chances for temporary failures that might pop up when sending. If we see a consistent set of failures, this will let us react more quickly.

Test Steps:
1. Spin up your local Docker instance
2. Let it run for about six minutes
3. Check the results in `receiver_connection_check_results`
4. Verify that there are no unexpected errors in the Docker scroll

## Changes
- Adds a new log table for the results of checking the remote receiver connections
- Adds logic to execute a check for all remote connections

## Checklist

### Testing

### Process

## Fixes
- #1153 
- #1062
